lib/capybara/selector/definition/button.rb

```diff
@@ -5,6 +5,7 @@
     input_btn_xpath = XPath.descendant(:input)[XPath.attr(:type).one_of('submit', 'reset', 'image', 'button')]
     btn_xpath = XPath.descendant(:button)
     image_btn_xpath = XPath.descendant(:input)[XPath.attr(:type) == 'image']
+    aria_btn_xpath = XPath.descendant[XPath.attr(:role).equals('button')]
```

Comment:
Any reason this can't just be part of `btn_xpath`? -  `btn_xpath = XPath.descendant(:button).union XPath.descendant[XPath.attr(:role).equals('button')]`